bin/release.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env sh
+
+release() {
+  
+  # Bump version using `yarn`.
+  yarn version --no-git-tag-version --new-version ${1:-patch}
+
+  # Get the new version number.
+  local version=`grep -m1 version package.json | awk -F: '{ print $2 }' | sed 's/[", ]//g'`
```

Comment:
A simpler approach (see https://github.com/uphold/uphold-sdk-javascript/pull/47):

```suggestion
  local version=`grep -m1 version package.json | cut -d '"' -f4`
```